operators/src/adapters/simple_raster_stacker.rs

```diff
@@ -0,0 +1,493 @@
+use crate::error::{AtLeastOneStreamRequired, Error};
+use crate::util::Result;
+use futures::ready;
+use futures::stream::Stream;
+use geoengine_datatypes::primitives::TimeInterval;
+use geoengine_datatypes::raster::RasterTile2D;
+use pin_project::pin_project;
+use snafu::ensure;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+/// Stacks the bands of the input raster streams to create a single raster stream with all the combined bands.
+///
+/// NOTE: ALL THE INPUT STREAMS MUST HAVE THE SAME TIME INTERVALS.
+///
+/// If the streams are not guaranteed to be temporally aligned, use the `RasterStackerAdapter` instead.
+#[pin_project(project = SimpleRasterStackerAdapterProjection)]
+pub struct SimpleRasterStackerAdapter<S> {
```

Comment:
Maybe call it "AlignedRasterStackerAdapter"?

Reply:
Is that really better? One might mistake it for the one aligning stuff.